.circleci/config.yml

```diff
@@ -117,7 +117,7 @@ workflows:
           xcode: "12.5.0"
           configuration: "Release"
 
-      - check-api-compatibility
+      # - check-api-compatibility
```

Comment:
Can you instead of workflow fix, change the actual `check-breaking-api` step? It would temporary disable checks for nightly build as well.

Reply:
Updated